### PR TITLE
WIP: use webcrypto-liner to support Safari 10

### PR DESCRIPTION
**Pull request by dannycoates**

### CODE REVIEW

File: frontend/src/download.js [hunk 1]
```diff
@@ -12,176 +12,186 @@ const Raven = window.Raven;
 
 $(document).ready(function() {
   //link back to homepage
```
Comment (pdehaan) [inline frontend/src/download.js]: This comment feels wrong now

Comment (dannycoates) [inline frontend/src/download.js]: hey man, WIP 😝

Comment (pdehaan) [inline frontend/src/download.js]: Free QA in Progress.

**Comment by pdehaan** [comment]
Ref: #411

**Comment by dannycoates** [comment]
There's one weird side effect on desktop Safari. After a successful download all the `target="_blank"` links *download the html* instead of opening a new tab. 🙈

File: views/download.handlebars [hunk 2]
```diff
@@ -35,5 +35,5 @@
     </div>
   </div>
 
-  <a class="send-new" data-l10n-id="sendYourFilesLink" target="_blank"></a>
+  <a class="send-new" data-l10n-id="sendYourFilesLink" href="/"></a>
```
Comment (pdehaan) [inline views/download.handlebars]: Didn't we have the `target="_blank"` there to prevent disrupting a download? #398

**Comment by pdehaan** [comment]
What is "public/cryptofill.js"? It looks vendored, but not sure if could be copied from **asmcrypto.js** or **webcrypto-liner** devDeps at build time (like l20n and babel-polyfill).

File: views/download.handlebars (hunk 2 above)
Comment (dannycoates) [inline views/download.handlebars]: See https://github.com/mozilla/send/blob/master/frontend/src/links.js#L7-L19 which gets called at the start of a download.

**Comment by dannycoates** [comment]
> What is "public/cryptofill.js"?

Its a custom build of those. I'll make that into a build step once we have something more sane to do it.

**Pull request merged**